CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# How to Contribute
+
+We'd love to accept your patches and contributions to this project. There are
+just a few small guidelines you need to follow.
+
+## Contributor License Agreement
+
+Contributions to this project must be accompanied by a Contributor License
+Agreement. You (or your employer) retain the copyright to your contribution;
+this simply gives us permission to use and redistribute your contributions as
+part of the project. Head over to <https://cla.developers.google.com/> to see
+your current agreements on file or to sign a new one.
+
+You generally only need to submit a CLA once, so if you've already submitted one
+(even if it was for a different project), you probably don't need to do it
+again.
+
+## Code reviews
+
+All submissions, including submissions by project members, require review. We
+use GitHub pull requests for this purpose. Consult
+[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
+information on using pull requests.
+
+## Community Guidelines
+
+This project follows
+[Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
+
+## Contributing code
+
+### Prerequisites
+Install:
+1. [Go](https://go.dev/) 1.18+
+2. [GoReleaser](https://goreleaser.com/) (Optional, only if you want reproducible builds)
+
+### Building
+
+#### Build using only `go`
+
+Run the following in the project directory:
+```shell
+$ go build ./cmd/osv-scanner/
```

Comment:
this command has no output, which is kinda confusing
I think it's useful to add a note it produces `./osv-scanner` as an executable
(no idea why is it directly in the source root, and not in `.gitignore`)

Reply:
Good point, added an explanation and put /osv-scanner into .gitignore